Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -506,15 +512,28 @@ theorem relIso_enum {α β : Type u} {r : α → α → Prop} {s : β → β →
 
 /-- The order isomorphism between ordinals less than `o` and `o.toType`. -/
 @[simps! -isSimp]
-noncomputable def enumIsoToType (o : Ordinal) : Set.Iio o ≃o o.toType where
+noncomputable def toType.mk {o : Ordinal} : Set.Iio o ≃o o.toType where
```

Comment:
This whole file is in a noncomputable section, so we don't need to write this out explicitly.
```suggestion
def toType.mk {o : Ordinal} : Set.Iio o ≃o o.toType where
```